docs/quick-start.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Quick Start
+sidebar_position: 10
+description: Quick start for running Inspektor Gadget.
+---
+
+This document contains quick instructions to use Inspektor Gadget in different
+environments and configurations.
+
+## Kubernetes
+
+### Kubectl Node Debug
+
+We can use [kubectl node debug](https://kubernetes.io/docs/tasks/debug/debug-cluster/kubectl-node-debug/) to run `ig` on a Kubernetes node:
+
+```bash
+$ kubectl debug --profile=sysadmin node/minikube -ti --image=ghcr.io/inspektor-gadget/ig -- ig run trace_exec:latest
+Creating debugging pod node-debugger-minikube-xw2sz with container debugger on node minikube.
+If you don't see a command prompt, try pressing enter.
+RUNTIME.CONTAINERN… COMM              PID        TID PCOMM            PPID ARGS      ER… TIMESTAMP             USER      LOGINUSER GROUP
+k8s_gadget_gadget-p gadgettra…     560811     560811 runc           560802 /bin/gad…     2024-07-31T20:20:56.… root      uid:4294… root
+k8s_gadget_gadget-p gadgettra…     560863     560863 container…     560853 /bin/gad…     2024-07-31T20:20:57.… root      uid:4294… root
+k8s_gadget_gadget-p gadgettra…     560923     560923 container…     560914 /bin/gad…     2024-07-31T20:21:01.… root      uid:4294… root
+k8s_gadget_gadget-p gadgettra…     560956     560956 container…     560946 /bin/gad…     2024-07-31T20:21:02.… root      uid:4294… root
+```
+
+For more information on how to use `ig` without installation on Kubernetes, please refer to the [ig documentation](./reference/ig.md#using-ig-with-kubectl-debug-node).
+
+
+### Deployed to the Cluster
+
+[krew](https://sigs.k8s.io/krew) is the recommended way to install
+`kubectl gadget`. You can follow the
+[krew's quickstart](https://krew.sigs.k8s.io/docs/user-guide/quickstart/)
+to install it and then install `kubectl gadget` by executing the following
+commands.
+
+```bash
+$ kubectl krew install gadget
+$ kubectl gadget deploy
```

Comment:
Makes sense. Since all these changes are actually mirrored from the Readme, I dropped this quick-start file from the PR. I'll open another PR to update the readme and sync with quick-start with it.